WTG.Analyzers.Test/RuleSetTest.cs

```diff
@@ -0,0 +1,43 @@
+using System;
+using System.IO;
+using System.Linq;
+using System.Xml.Linq;
+using Microsoft.CodeAnalysis;
+using NUnit.Framework;
+
+namespace WTG.Analyzers.Test
+{
+	[TestFixture]
+	class RuleSetTest
+	{
+		[Test]
+		public void WarnAll()
+		{
+			var rulesetPath = Path.Combine(TestContext.CurrentContext.TestDirectory, "..", "..", "WTG.Analyzers", "build", "WarnAll.ruleset");
```

Comment:
Consider wrapping in `Path.GetFullPath` - makes it easier to debug.

Reply:
Done.